scipy/spatial/_spherical_voronoi.py

```diff
@@ -206,6 +228,8 @@ def __init__(self, points, radius=None, center=None):
         """
 
         self.points = points
+        if pdist(self.points).min() <= threshold:
```

Comment:
Would be better to have the rhs be `threshold * radius`, so that `threshold` is independent of scaling.